python/vulns_per_assets/export_assets.py

```diff
@@ -0,0 +1,165 @@
+# Does an asset export and for each asset, collects the vulnerability information.
+# A gzip file in the form of assets_xxxx.gz and a unzip file assets_xxxx are left
+# around.
+#
+# The script has one optional parameter, the search ID.  If specified, it is assumed
+# that you know the search ID from a previous search.
+
+import os
+import sys
+import time
+import requests
+import json
+import gzip
+import shutil
+import math
+
```

Comment:
A lot of these functions appear to be duplicated from the export_asset_vulns script. Would it be worth considering a function library to which both scripts can refer back?

Reply:
Good suggestion.  Since these are blog examples, my thought was to keep each example contained.  I will consider this for next time.